Can't run the SSG after installation

I got this error after following the instructions in the README file.

```bash
OP:~ admin$ git clone https://github.com/oliver-pham/cmd-ssg.git
Cloning into 'cmd-ssg'...
remote: Enumerating objects: 1376, done.
remote: Counting objects: 100% (1376/1376), done.
remote: Compressing objects: 100% (1148/1148), done.
remote: Total 1376 (delta 208), reused 1343 (delta 186), pack-reused 0
Receiving objects: 100% (1376/1376), 30.09 MiB | 15.79 MiB/s, done.
Resolving deltas: 100% (208/208), done.
warning: unable to access '/Users/admin/.config/git/attributes': Permission denied
OP:~ admin$
OP:~ admin$ cd cmd-ssg/
OP:cmd-ssg admin$
OP:cmd-ssg admin$ npm install -g .

added 1 package, and audited 3 packages in 1s

found 0 vulnerabilities
OP:cmd-ssg admin$ ssgy -h
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'chalk'
Require stack:
- /Users/admin/cmd-ssg/bin/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/Users/admin/cmd-ssg/bin/index.js:4:15)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/admin/cmd-ssg/bin/index.js' ]
}
```

--- a/README.md
+++ b/README.md
@@ -6,7 +6,7 @@ deliverable 0.1 for OSD600 open source course at seneca
 
 ```
   VERSION    : 0.1.0
-  Use        : Process input .txt/.md files into generated .html files.
+  Use        : Process input .txt files into generated .html files.
 ```
 
 ## Prerequiste for development
@@ -15,7 +15,10 @@ deliverable 0.1 for OSD600 open source course at seneca
    "node": ">=16.9.1",
    "npm": ">=7.23.0"
 
-   ```
+   if you are developing,
+   run npm install
+
+```
 
 ## Screen Shot
 
@@ -27,17 +30,11 @@ deliverable 0.1 for OSD600 open source course at seneca
 
 ```
 git clone <this repo>
-   Linux: npm install
-          sudo npm install -g .
-   Windows: npm install
-            npm install -g .
-   Macs:  <not fully supported>
-
+npm install -g . or npm install if you are just testing
 ssgy <command option>
 
-   Example:
-      [x] ssgy -i examples
-      [x] ssgy -i examples/md -s sample_css/new.css
+Example:
+ssgy -i examples
 ```
 
 ## FEATURES :
@@ -58,18 +55,9 @@ ssgy <command option>
 13. parse a title from your input files
 14. adding -s stylesheet option
 15. making it easy on the eyes =) checkout my cmd-ssg -h
-16. generate static files for Markdown files
-   - [x] Parse Heading syntax
-   - [x] Parse Italics & Bold syntax
 
-# Contributing
-
-Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 ```
 Special Thanks  : Kevan Yang
 Markdown Feature: Oliver Pham
 Author          : Eugene Chung
 ```
-
-# License
-[MIT](LICENSE)

